devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -193,9 +189,7 @@ public void execute(Task test) {
                                     .plus(testSourceSet.getOutput()));
 
                     // create a custom configuration for devmode
-                    configurations.create(DEV_MODE_CONFIGURATION_NAME).extendsFrom(
-                            configurations.getByName(JavaPlugin.COMPILE_ONLY_CONFIGURATION_NAME),
-                            configurations.getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME));
+                    configurations.create(DEV_MODE_CONFIGURATION_NAME);
```

Comment:
What does this configuration contain now?

Reply:
only dependency with the `quarkusDev` scope. So in most case nothing.